ush/load_ufsda_modules.sh

```diff
@@ -1,7 +1,8 @@
 #! /usr/bin/env bash
 
 ###############################################################
-if [[ "${DEBUG_WORKFLOW:-NO}" == "NO" ]]; then
+export DEBUG_WORKFLOW="${DEBUG_WORKFLOW:-NO}"
```

Comment:
is this a lint thing?

Reply:
@CoryMartin-NOAA , nope, that was an issue before you added a default to the other places where `DEBUG_WORKFLOW` is used. I'll remove that line.